src/plugins/workspace/server/integration_tests/routes.test.ts

```diff
@@ -696,4 +671,70 @@ describe('workspace service api integration test when savedObjects.permission.en
       ).toEqual({ write: { users: ['foo'] } });
     });
   });
+
+  describe('Saved Object Management APIs', () => {
```

Comment:
Shall we move these integration tests under `src/plugins/saved_objects_management` folder? It was testing API in saved objects management instead of workspace routes.